### ISSUE TITLE
difficult to use sge log file names

### ISSUE THREAD

**Issue by dgordon562** [PacificBiosciences/FALCON#426]
Hi, Chris,

Would it be possible to return to the sge log file names of days gone by in which the m_(name) or job_(name) was the initial part of the log file name?  Every falcon assembly I do requires reading some (many) log files and it would make this process so much easier...  Tab completion would allow finding the correct log file, even if there are arbitrary characters thrown in later in the filename.

(I heard one guy joke about trying to debug a program in which each variable was a 30 character arbitrary unique alphanumeric string...grin)

Thanks!
David

**Comment by pb-cdunn** [comment]
I'm open to anything. But let's discuss first.

What's wrong with the `pwatcher.dir/std*` files? That's a symlink, but it appears in the actual run-directory.

(I still have to separate the preads jobs into sub-dirs. Soon.)

**Comment by pb-jchin** [comment]
yes, @pb-cdunn's design to collect log inside each task is good. For example, you can use
```
cat 0-rawreads/job_000c/pwatcher.dir/stdout
```
